README.md

```diff
@@ -118,9 +126,9 @@ You can run the program with `--help` parameter to see more info:
     ./programs/witness_node/witness_node --help
 
 
-### Run the command-line wallet software
+### Command-line Wallet Software
```

Comment:
This section and the "Run the node software" were on the same level. Now it reads different.

Reply:
Sorry I don't understand what you mean here.

Reply:
It was
```
### Run the node software
...
### Run the command-line wallet software
...
```
It is now
```
### Run

**Run Node Software:**

...

### Command-line Wallet Software

...
```